athlinks-schema.ttl

```diff
@@ -13,6 +13,32 @@
   rdfs:label "The Athlinks Schema vocabulary".
 
 ### CLASSES
+athlinks:TerrainSpecifics a rdfs:Class;
+  rdfs:comment "Specifics about course terrain such as altitude or grade." ;
+  rdfs:label "Terrain Specifics" ;
+  rdfs:subClassOf schema:Landform ;
+  #members:
+  #athlinks:elevUnit;
+  #athlinks:altitude;
+
+athlinks:Directions a rdfs:Class;
+  rdfs:comment "Directions to an event or race." ;
+  rdfs:label "Directions" ;
+  rdfs:subClassOf schema:Text ;
+
+athlinks:AgeReferenceDate a rdfs:Class;
+  rdfs:comment "When event has a minimum age, this is the date by which they must be that age." ;
+  rdfs:label "Age Reference Date" ;
+  rdfs:subClassOf schema:Date ;
+
+athlinks:Timer a rdfs:Class;
+  rdfs:comment "A person or organization who times events." ;
```

Comment:
I don't think we need athlinks:Timer. 
Isn't a "timer" a person or organization who times events? Thus:
```turtle
<some:foo> a schema:Person;
         athlinks:timerOf <some:event>.
```
Would mean that <some:foo> is a timer. So I don't see what creating athlinks:Timer does for us.